docs/tutorials/python/json_schema.md

```diff
@@ -0,0 +1,124 @@
+
+```python
+{!docs/tutorials/python/tutorial_scripts/json_schema.py!lines=1-20}
+```
+
+## Step 2: Take a look at the constants and structure of the JSON schema
+
+For the purpose of this tutorial, an organization named "myUniqueAlzheimersResearchOrgTutorial" has been created, and within it, a schema named clinicalObservations has been registered.
+
+```python
+{!docs/tutorials/python/tutorial_scripts/json_schema.py!lines=23-49}
+```
+
+Derived annotations allow you to define default values for annotations based on schema rules, ensuring consistency and reducing manual input errors. As you can see here, you could use derived annotations to prescribe default annotation values. Please read more about derived annotations [here](https://help.synapse.org/docs/JSON-Schemas.3107291536.html#JSONSchemas-DerivedAnnotations).
+
+
+## Step 3: Retrieve test organization
+Next, retrieve an organization:
+```python
+{!docs/tutorials/python/tutorial_scripts/json_schema.py!lines=51-54}
+```
+
+## Step 4: Bind the JSON schema to the folder
+After retrieving the organization, you can now bind your json schema to a test folder. When you bind a JSON Schema to a project or folder, then all items inside of the project or folder will inherit the schema binding, unless the item has a schema bound to itself.
+
+When you bind the schema, you may also include the boolean property `enable_derived_annos` to have Synapse automatically calculate derived annotations based on the schema:
+
+```python
+{!docs/tutorials/python/tutorial_scripts/json_schema.py!lines=56-63}
+```
+
+## Step 5: Retrieve the Bound Schema
+Next, we can retrieve the bound schema:
+```python
+{!docs/tutorials/python/tutorial_scripts/json_schema.py!lines=65-67}
+```
+
+## Step 6: Add Invalid Annotations to the Folder and Store
+Try adding invalid annotations to your folder: This step and the step below demonstrate how the system handles invalid annotations and how the schema validation process works.
+```python
+{!docs/tutorials/python/tutorial_scripts/json_schema.py!lines=70-76}
+```
+
+In the synapse web UI, you could also see your invalid annotations being marked by a yellow label similar to this:
+
+![json_schema](./tutorial_screenshots/jsonschema_folder.png)
+
+
+## Step 7: Validate Folder Against the Schema
+Try validating the folder. You should be able to see messages related to invalid annotations.
+```python
+{!docs/tutorials/python/tutorial_scripts/json_schema.py!lines=79-81}
+```
+
+## Step 8: Create a File with Invalid Annotations and Upload It
+Try creating a test file locally and store the file in the folder that we created earlier. Then, try adding invalid annotations to a file. This step demonstrates how the files inside a folder also inherit the schema from the parent entity.
+```python
+{!docs/tutorials/python/tutorial_scripts/json_schema.py!lines=84-111}
+```
+
+In the synapse web UI, you could also see your invalid annotations being marked by a yellow label similar to this:
```

Comment:
Similar comments to:
https://github.com/Sage-Bionetworks/synapsePythonClient/pull/1205/files#r2150665500

Show programmatic validation first (And console logs) and then web UI